asyncom/om.py

```diff
@@ -21,25 +25,25 @@ def map_result(v):
     return map_result
 
 
-class OMQuery(Query):
-    def __init__(self, entities, database=None,
+class OMQuery(Query, Generic[T]):
+    def __init__(self, entity: T, database=None,
                  mapper_factory=default_mapper_factory):
         self.__db = database
         self._all = None
         self._mapper_factory = mapper_factory
-        super().__init__(entities, session=None)
+        super().__init__([entity], session=None)
```

Comment:
You'll notice I changed the list into `entities` to `[entity]`. The reason is that AFAIK, `OMQuery` doesn't support multiple entities and always use the first one (https://github.com/vinissimus/asyncom/pull/6/files#diff-0c5e3f164e3f861aefdd3e518a50d0ceR19). Is this change ok?